Source/WebCore/platform/graphics/PathSegment.cpp

```diff
@@ -39,75 +39,62 @@ PathSegment::PathSegment(Data&& data)
 
 FloatPoint PathSegment::calculateEndPoint(const FloatPoint& currentPoint, FloatPoint& lastMoveToPoint) const
 {
-    return WTF::switchOn(m_data,
```

Comment:
Ah that's just because I had to upload the PR with the dependent commit in there. If you review the individual commit it's fine. (Though the dependent commit has now landed, so I should rebase.)